pxr/usdImaging/usdImaging/cameraAdapter.cpp

```diff
@@ -259,9 +259,33 @@ UsdImagingCameraAdapter::Get(UsdPrim const& prim,
         cam.GetShutterCloseAttr().Get(&vShutterClose, time); // conversion n/a
         return vShutterClose;
     } else if (key == HdCameraTokens->exposure) {
-        VtValue v;
-        cam.GetExposureAttr().Get(&v, time); // conversion n/a
-        return v;
+        // we hijack the "exposure" token here in order to pass through the calculated
```

Comment:
@anderslanglands This part worries me a bit, as it has the potential to be confusing forever.  If I swap us over to just transporting the attributes as named, compute the exposure scale in HdCamera, and update our renderers to look for exposureScale and fallback to exposure, would you have any major objections?  It would be a more manual upgrade process to get the benefit of this new schema, but the transport side wouldn't be confusing, and if we add a good imaging unit test that seems like it would help with compliance.

A similar variant of that is to keep the GetExposureScale here but transport it as "exposureScale", and otherwise do what I mentioned above.

Reply:
Hi Tom! No I wouldn't have major objections. I found it "nice" that it all just fed through automatically but I understand your concerns about long-term confusion just for the sake of a little ease now, particularly when a bunch of renderers don't actually pay attention to exposure anyway.

I can't find your other comment on here about logarithmic vs linear now... but in reference to that, personally I think linear makes more sense. I did have to convert it to logarithmic in the adapter just to make the cute "hijack exposure" trick work, so if we move away from that I would keep it linear as there's no good reason for it to be logarithmic and it saves a pow2 on the renderer side.